python/nx-cugraph/nx_cugraph/classes/graph.py

```diff
@@ -157,6 +159,19 @@ def from_coo(
                     f"(got {new_graph.dst_indices.dtype.name})."
                 )
             new_graph.dst_indices = dst_indices
+
+        # If the graph contains isolates, plc.SGGraph() must be passed a value
+        # for vertices_array that contains every vertex ID, since the
+        # src/dst_indices arrays will not contain IDs for isolates. Create this
+        # only if needed. Like src/dst_indices, the _node_ids array must be
+        # maintained for the lifetime of the plc.SGGraph
+        all_node_ids = cp.arange(new_graph._N, dtype=index_dtype)
+        isolates = cp.setdiff1d(
+            cp.setdiff1d(all_node_ids, new_graph.src_indices), new_graph.dst_indices
+        )
```

Comment:
Thanks for the code comments.

I think we could also use `isolates = nxcg.algorithms.isolate._isolates(new_graph)` here (not sure which way is faster--they're both probably pretty fast).

Reply:
I'm using `isolates = nxcg.algorithms.isolate._isolates(new_graph)` to reduce duplication and since it's a bit faster in my experiments.